src/main/java/com/odysseusinc/athena/api/v1/controller/VocabularyController.java

```diff
@@ -320,6 +315,14 @@ public ResponseEntity notifyAboutUpdates(
         final AthenaUser user = userService.getUser(principal);
         List<Notification> notifications = vocabularyService.getNotifications(user.getId());
         List<VocabularyDTO> vocabularyDTOs = converterUtils.convertList(notifications, VocabularyDTO.class);
-        return new ResponseEntity<>(vocabularyDTOs, HttpStatus.OK);
+        return ResponseEntity.ok(vocabularyDTOs);
+    }
+
+    @GetMapping(value = "/releaseVersion")
```

Comment:
The whole application utilizes hyphens in URLs and you now introduce camel case. Why?